library/src/data/index.js

```diff
@@ -1,24 +1,38 @@
 window.CoNDeT.data = {
-  State: (function () {
-    function constructor() {
-      this.state = [];
-      this.subscribers = [];
-    }
+  FileReaderWriter:{
+  
+
+  readFromFile =function (cb) {
```

Comment:
Już wiem dla czego prietteir nie działał. Bo to object literal, czyli `name: value` a nie `name = value`. Biedny nie wiedział co zrobić. Popraw to i tak samo dla `saveToFile`. Dodatkowo wygląda, jakby usuwało `State`. Więc sprawdź, czy go przez przypadek nie nadpisałeś